src/features/device-state/routes/state.ts

```diff
@@ -287,6 +334,30 @@ const stateQuery = _.once(() =>
 						},
 					},
 				},
+				should_be_managed_by__release: {
+					$select: releaseSelect,
+					$expand: {
+						has__tag_key: releaseExpand.has__tag_key,
+						contains__image: releaseExpand.contains__image,
+						belongs_to__application: {
```

Comment:
This is just extending the default expand right? In that case it'd be nicer to do:
```suggestion
					$expand: {
						...releaseExpand,
						belongs_to__application: {
```